packages/myst-execute/src/types.ts

```diff
@@ -0,0 +1,26 @@
+import type * as nbformat from '@jupyterlab/nbformat';
+import type { PartialJSONObject } from '@lumino/coreutils';
+
+export interface IExpressionOutput {
+  status: 'ok';
+  data: nbformat.IMimeBundle;
+  metadata: PartialJSONObject;
+}
+
+export interface IExpressionError {
+  status: 'error';
+  /**
+   * Exception name
+   */
+  ename: string;
+  /**
+   * Exception value
+   */
+  evalue: string;
+  /**
+   * Traceback
+   */
+  traceback: string[];
+}
+
+export type IExpressionResult = IExpressionError | IExpressionOutput;
```

Comment:
@rowanc1 not sure on where these types should go — they're very relevant to Jupyter, so putting them here makes sense. Equally, should we be able to talk about expression results _without_ needing to pull in execution? I suppose with type imports being compile-time only, it's not the end of the world. But, I'm wondering if we need a second package?

Reply:
The `myst-common` package is the place for these:

https://github.com/executablebooks/mystmd/blob/main/packages/myst-common/src/types.ts